src/main/kotlin/org/rust/lang/core/completion/RsCompletionContributor.kt

```diff
@@ -18,6 +18,7 @@ class RsCompletionContributor : CompletionContributor() {
         extend(CompletionType.BASIC, RsDeriveCompletionProvider.elementPattern, RsDeriveCompletionProvider)
         extend(CompletionType.BASIC, AttributeCompletionProvider.elementPattern, AttributeCompletionProvider)
         extend(CompletionType.BASIC, RsMacroCompletionProvider.elementPattern, RsMacroCompletionProvider)
+        extend(CompletionType.BASIC, RsPrimitiveTypeCompletionProvider.elementPattern, RsPrimitiveTypeCompletionProvider)
```

Comment:
Maybe it would be better to use this provider before `RsCommonCompletionProvider`?